.gitignore

```diff
@@ -1,3 +1,6 @@
 workspace/
 .ipynb_checkpoints/
 financespy/__pycache__
+financespy.egg-info/
```

Comment:
financespy.egg-info, pycache and .ipynb_checkpoints should not go to version control, please delete and add to .gitignore